### ISSUE TITLE
Improve your README and share it :)

### ISSUE THREAD

**Issue by tunnckoCore** [AndiChiou/AMA#6]
Heya Andi.
Welcome to GitHub. If you're beginner with Git and GitHub you can read [this small git guide](http://rogerdudler.github.io/git-guide/).

And if you're planing to use actively GitHub and sharing your work you can send a PR to @sindresorhus's AMAs repository to add your AMA to [the big list of other amazing AMAs](https://github.com/sindresorhus/amas). I'm near the top, but mostly in the middle haha. It was just explosion when he created it.

**Comment by AndiChiou** [comment]
Yes. I am planning to use GitHub extensively. Thanks for your advice. It was helpful.
Your AMA looks great with those batches. 
I think this will be my first pull request 😃 .

**Comment by sshiza-490** [comment]
Hy love you dear
